### ISSUE TITLE
build.py generates a lot of warnings for dart: libraries

### ISSUE THREAD

**Issue by karlklose** [dart-lang/sdk#31420]
```[3573/3626] ACTION //utils/dartdevc:dartdevc_sdk(//build/toolchain/linux:clang_x64)
[warning] The final variable 'length' must be initialized. (dart:_interceptors/js_string.dart, line 15, col 9)
[warning] The final variable 'origin' must be initialized. (dart:html, line 193, col 3)
[warning] The final variable 'origin' must be initialized. (dart:html, line 863, col 3)
[warning] The final variables 'form', 'labels' and '3' more must be initialized. (dart:html, line 1679, col 3)
[warning] The final variable 'options' must be initialized. (dart:html, line 8979, col 3)
[warning] The final variables '_attributes', '_childElementCount' and '19' more must be initialized. (dart:html, line 13239, col 3)
[warning] The final variables 'elements', 'form' and '4' more must be initialized. (dart:html, line 17115, col 3)
[warning] The final variable 'length' must be initialized. (dart:html, line 18065, col 3)
[warning] The final variables '_get_contentWindow' and 'sandbox' must be initialized. (dart:html, line 20777, col 3)
[warning] The final variables 'complete', 'currentSrc' and '2' more must be initialized. (dart:html, line 20984, col 3)
[warning] The final variables '_get_valueAsDate', 'entries' and '6' more must be initialized. (dart:html, line 21149, col 3)
[warning] The final variables 'form', 'labels' and '4' more must be initialized. (dart:html, line 22359, col 3)
[warning] The final variables 'control' and 'form' must be initialized. (dart:html, line 22506, col 3)
[warning] The final variable 'form' must be initialized. (dart:html, line 22545, col 3)
[warning] The final variables 'import', 'relList' and '2' more must be initialized. (dart:html, line 22634, col 3)
[warning] The final variable 'areas' must be initialized. (dart:html, line 22797, col 3)
[warning] The final variables 'audioDecodedByteCount', 'audioTracks' and '16' more must be initialized. (dart:html, line 23098, col 3)
[warning] The final variable 'labels' must be initialized. (dart:html, line 24692, col 3)
[warning] The final variables 'baseUri', 'childNodes' and '11' more must be initialized. (dart:html, line 26236, col 3)
[warning] The final variables 'form', 'validationMessage' and '2' more must be initialized. (dart:html, line 27188, col 3)
[warning] The final variables 'form' and 'index' must be initialized. (dart:html, line 27370, col 3)
[warning] The final variables 'form', 'htmlFor' and '5' more must be initialized. (dart:html, line 27424, col 3)
[warning] The final variables 'labels' and 'position' must be initialized. (dart:html, line 28986, col 3)
[warning] The final variables 'form', 'labels' and '4' more must be initialized. (dart:html, line 30915, col 3)
[warning] The final variable 'sheet' must be initialized. (dart:html, line 33390, col 3)
[warning] The final variable 'cellIndex' must be initialized. (dart:html, line 33655, col 3)
[warning] The final variables '_rows' and '_tBodies' must be initialized. (dart:html, line 33774, col 3)
[warning] The final variables '_cells', 'rowIndex' and '1' more must be initialized. (dart:html, line 33891, col 3)
[warning] The final variable '_rows' must be initialized. (dart:html, line 33960, col 3)
[warning] The final variable 'content' must be initialized. (dart:html, line 34004, col 3)
[warning] The final variables 'form', 'labels' and '5' more must be initialized. (dart:html, line 34093, col 3)
[warning] The final variables 'readyState' and 'track' must be initialized. (dart:html, line 35153, col 3)
[warning] The final variables 'decodedFrameCount', 'droppedFrameCount' and '2' more must be initialized. (dart:html, line 36060, col 3)
[warning] The final variable 'sourceCapabilities' must be initialized. (dart:html, line 46038, col 3)
[warning] The final variables 'href' and 'target' must be initialized. (dart:svg, line 56, col 3)
[warning] The final variables 'requiredExtensions', 'requiredFeatures' and '2' more must be initialized. (dart:svg, line 512, col 3)
[warning] The final variables 'cx', 'cy' and '1' more must be initialized. (dart:svg, line 583, col 3)
[warning] The final variable 'clipPathUnits' must be initialized. (dart:svg, line 620, col 3)
[warning] The final variables 'cx', 'cy' and '2' more must be initialized. (dart:svg, line 719, col 3)
[warning] The final variables 'height', 'in1' and '6' more must be initialized. (dart:svg, line 765, col 3)
[warning] The final variables 'height', 'in1' and '6' more must be initialized. (dart:svg, line 858, col 3)
[warning] The final variables 'height', 'in1' and '4' more must be initialized. (dart:svg, line 947, col 3)
[warning] The final variables 'height', 'in1' and '10' more must be initialized. (dart:svg, line 999, col 3)
[warning] The final variables 'bias', 'divisor' and '15' more must be initialized. (dart:svg, line 1107, col 3)
[warning] The final variables 'diffuseConstant', 'height' and '8' more must be initialized. (dart:svg, line 1228, col 3)
[warning] The final variables 'height', 'in1' and '8' more must be initialized. (dart:svg, line 1305, col 3)
[warning] The final variables 'azimuth' and 'elevation' must be initialized. (dart:svg, line 1401, col 3)
[warning] The final variables 'height', 'result' and '3' more must be initialized. (dart:svg, line 1444, col 3)
[warning] The final variables 'height', 'in1' and '6' more must be initialized. (dart:svg, line 1637, col 3)
[warning] The final variables 'height', 'href' and '5' more must be initialized. (dart:svg, line 1710, col 3)
[warning] The final variables 'height', 'result' and '3' more must be initialized. (dart:svg, line 1777, col 3)
[warning] The final variable 'in1' must be initialized. (dart:svg, line 1833, col 3)
[warning] The final variables 'height', 'in1' and '7' more must be initialized. (dart:svg, line 1867, col 3)
[warning] The final variables 'dx', 'dy' and '6' more must be initialized. (dart:svg, line 1947, col 3)
[warning] The final variables 'x', 'y' and '1' more must be initialized. (dart:svg, line 2015, col 3)
[warning] The final variables 'height', 'in1' and '9' more must be initialized. (dart:svg, line 2062, col 3)
[warning] The final variables 'limitingConeAngle', 'pointsAtX' and '6' more must be initialized. (dart:svg, line 2144, col 3)
[warning] The final variables 'height', 'in1' and '4' more must be initialized. (dart:svg, line 2211, col 3)
[warning] The final variables 'baseFrequencyX', 'baseFrequencyY' and '9' more must be initialized. (dart:svg, line 2272, col 3)
[warning] The final variables 'filterUnits', 'height' and '5' more must be initialized. (dart:svg, line 2376, col 3)
[warning] The final variables 'height', 'width' and '2' more must be initialized. (dart:svg, line 2479, col 3)
[warning] The final variables 'farthestViewportElement', 'nearestViewportElement' and '4' more must be initialized. (dart:svg, line 2574, col 3)
[warning] The final variables 'height', 'href' and '4' more must be initialized. (dart:svg, line 2648, col 3)
[warning] The final variables 'x1', 'x2' and '2' more must be initialized. (dart:svg, line 2882, col 3)
[warning] The final variables 'x1', 'x2' and '2' more must be initialized. (dart:svg, line 2923, col 3)
[warning] The final variables 'markerHeight', 'markerUnits' and '7' more must be initialized. (dart:svg, line 2964, col 3)
[warning] The final variables 'height', 'maskContentUnits' and '7' more must be initialized. (dart:svg, line 3059, col 3)
[warning] The final variable 'pathLength' must be initialized. (dart:svg, line 3343, col 3)
[warning] The final variables 'height', 'href' and '11' more must be initialized. (dart:svg, line 3385, col 3)
[warning] The final variables 'animatedPoints' and 'points' must be initialized. (dart:svg, line 3550, col 3)
[warning] The final variables 'animatedPoints' and 'points' must be initialized. (dart:svg, line 3583, col 3)
[warning] The final variables 'cx', 'cy' and '4' more must be initialized. (dart:svg, line 3694, col 3)
[warning] The final variables 'height', 'rx' and '4' more must be initialized. (dart:svg, line 3773, col 3)
[warning] The final variable 'href' must be initialized. (dart:svg, line 3822, col 3)
[warning] The final variable 'gradientOffset' must be initialized. (dart:svg, line 3890, col 3)
[warning] The final variable 'sheet' must be initialized. (dart:svg, line 4022, col 3)
[warning] The final variables 'ownerSvgElement' and 'viewportElement' must be initialized. (dart:svg, line 4536, col 3)
[warning] The final variables 'currentTranslate', 'currentView' and '8' more must be initialized. (dart:svg, line 4880, col 3)
[warning] The final variables 'preserveAspectRatio' and 'viewBox' must be initialized. (dart:svg, line 5086, col 3)
[warning] The final variables 'lengthAdjust' and 'textLength' must be initialized. (dart:svg, line 5160, col 3)
[warning] The final variables 'href', 'method' and '2' more must be initialized. (dart:svg, line 5261, col 3)
[warning] The final variables 'dx', 'dy' and '3' more must be initialized. (dart:svg, line 5323, col 3)
[warning] The final variables 'height', 'href' and '3' more must be initialized. (dart:svg, line 5622, col 3)
[warning] The final variables 'preserveAspectRatio', 'viewBox' and '1' more must be initialized. (dart:svg, line 5669, col 3)
[warning] The final variables 'gradientTransform', 'gradientUnits' and '2' more must be initialized. (dart:svg, line 5827, col 3)```

**Comment by jensjoha** [comment]
Introduced in 68dc77f456edaee2341fc14f7e02d0e0c630838a.

**Comment by kmillikin** [comment]
> `[warning] The final variables 'form', 'labels' and '3' more must be initialized. (dart:html, line 1679, col 3)`

The single quotes around 3 are a very nice touch.

**Comment by jensjoha** [comment]
This appears to have been fixed in 37cd5e33ce3fe31428ac69f7250bdb421f4349db.

**Comment by munificent** [comment]
Yup, sorry for the noise!
